examples/reference/elements/matplotlib/Spikes.ipynb

```diff
@@ -57,9 +58,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%opts Points (color='red')\n",
     "ys = np.random.rand(50)\n",
-    "hv.Points((xs, ys)) * hv.Spikes(xs)"
+    "hv.Points((xs, ys)).opts(color='red') * hv.Spikes(xs)"
```

Comment:
Should be split out either:

```python
points = hv.Points((xs, ys))
spikes = hv.Spikes(xs)
points.opts(color='red') * spikes
```

or:

```python
points = hv.Points((xs, ys))
spikes = hv.Spikes(xs)
(points * spikes).opts(opts.Points(color='red')
```

Reply:
Since we're only styling one of the elements here I still find ``points.opts(color='red') * spikes`` most readable.